### PR TITLE
Update review model

### PR DESCRIPTION
**Pull request by jamiesprax**

### CODE REVIEW

File: src/main/java/org/atlasapi/media/entity/ReviewType.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return this.equals(FOTD_REVIEW) ? "fotd_transmit_date" : "";
+    }
+
+    @Nullable
+    public static ReviewType fromKey(String key) {
+        return Arrays.stream(ReviewType.values())
+                .filter(reviewType -> key.equalsIgnoreCase(reviewType.name()))
+                .findFirst().orElse(null);
+    }
+}
```
Comment (dragospitica) [inline src/main/java/org/atlasapi/media/entity/ReviewType.java]: new line

File: src/main/java/org/atlasapi/media/entity/ReviewType.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    @Nullable
+    public static ReviewType fromKey(String key) {
+
+        return Arrays.stream(ReviewType.values())
+                .filter(reviewType -> reviewType.name().equalsIgnoreCase(key))
+                .findFirst().orElse(null);
+    }
+}
```
Comment (AdamDM) [inline src/main/java/org/atlasapi/media/entity/ReviewType.java]: nl

File: src/main/java/org/atlasapi/media/entity/ReviewType.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package org.atlasapi.media.entity;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+
+public enum ReviewType {
```
Comment (AdamDM) [inline src/main/java/org/atlasapi/media/entity/ReviewType.java]: why not make this enum inline in the Review class?

Comment (jamiesprax) [inline src/main/java/org/atlasapi/media/entity/ReviewType.java]: It's used by the the simple & complex model

**Pull request merged**